consonance-webservice/src/main/java/io/swagger/wes/api/impl/Ga4ghApiServiceImpl.java

```diff
@@ -0,0 +1,414 @@
+import io.swagger.models.auth.In;
+import io.swagger.wes.api.*;
+import io.swagger.wes.model.*;
+
+import io.consonance.arch.beans.Job;
+
+import io.swagger.wes.model.Ga4ghWesServiceInfo;
+import io.swagger.wes.model.Ga4ghWesWorkflowListResponse;
+import io.swagger.wes.model.Ga4ghWesWorkflowLog;
+import io.swagger.wes.model.Ga4ghWesWorkflowRequest;
+import io.swagger.wes.model.Ga4ghWesWorkflowRunId;
+import io.swagger.wes.model.Ga4ghWesWorkflowStatus;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import io.swagger.wes.api.NotFoundException;
+
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Stream;
+import static java.nio.file.StandardOpenOption.*;
+
+
+
+import org.apache.commons.configuration.HierarchicalINIConfiguration;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.commons.io.FileUtils;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+
+import io.consonance.webservice.ConsonanceWebserviceConfiguration;
+import io.consonance.webservice.core.ConsonanceUser;
+import io.consonance.webservice.resources.OrderResource;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+import javax.validation.constraints.*;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.commons.validator.routines.UrlValidator;
+import scala.Int;
+
+import static io.dockstore.client.cli.ArgumentUtility.kill;
+
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaJerseyServerCodegen", date = "2017-09-15T17:06:31.319-07:00")
+public class Ga4ghApiServiceImpl extends Ga4ghApiService {
+    private static final ObjectMapper OBJECT_MAPPER;
+
+    static {
+        OBJECT_MAPPER = new ObjectMapper();
+        OBJECT_MAPPER.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+        OBJECT_MAPPER.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
+        OBJECT_MAPPER.enable(SerializationFeature.INDENT_OUTPUT);
+        OBJECT_MAPPER.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+    }
+
+    private static final Logger LOG = LoggerFactory.getLogger(Ga4ghApiServiceImpl.class);
+
+    private HierarchicalINIConfiguration settings = null;
+    private static ConsonanceWebserviceConfiguration config;
+    private static OrderResource orderResource;
+    private static Gson gson = new Gson();
+
+    public static void setConfig(ConsonanceWebserviceConfiguration config) {
+        Ga4ghApiServiceImpl.config = config;
+    }
+
+    public static void setOrderResource(OrderResource orderResource) {
+        Ga4ghApiServiceImpl.orderResource = orderResource;
+    }
+
+    // Cancels a running workflow by providing the id of the workflow
+    @Override
+    public Response cancelJob(String workflowId, ConsonanceUser user) throws NotFoundException {
+
```

Comment:
Maybe throw a `new UnsupportedOperationException()` or similar instead of claiming success